doc/environment.md

```diff
@@ -36,7 +36,7 @@ Name                            | Description
 `INCUS_EXEC_PATH`               | Full path to the Incus binary (used when forking subcommands)
 `INCUS_IDMAPPED_MOUNTS_DISABLE` | Disable idmapped mounts support (useful when testing traditional UID shifting)
 `INCUS_LXC_TEMPLATE_CONFIG`     | Path to the LXC template configuration directory
-`INCUS_OVMF_PATH`               | Path to an OVMF build including `OVMF_CODE.fd` and `OVMF_VARS.ms.fd`
+`INCUS_EFI_PATH`                | Path to EFI firmware build including `*_CODE.fd` and `*_VARS.fd`
```

Comment:
I'd probably use `INCUS_EDK2_PATH` instead to be a bit more specific about what we're using.

Reply:
Based on your other comment about the ed2k driver would prefer `edk2` be used throughout the code instead of the more generic `efi`?

Reply:
Yeah, I think so, especially as EFI is misleading anyway given that what we're using today is UEFI, not the old EFI (which I had to deal with a long time ago ;)).